js-exercises/telephone-check/telephoneCheck.js

```diff
@@ -0,0 +1,6 @@
+function telephoneCheck(phoneNumber) {
+  const phoneNumberRegex = /^(1\s?)?((\([0-9]{3}\))|[0-9]{3})[\s\-]?[\0-9]{3}[\s\-]?[0-9]{4}$/;
```

Comment:
what do think about giving **validateNo**  inspite of phoneNumberRegex ? I think validateNo is a good option